backend/src/main/java/ca/bc/gov/hlth/mohums/security/SecurityConfig.java

```diff
@@ -35,9 +35,12 @@ public SecurityWebFilterChain securityWebFilterChain(final ServerHttpSecurity ht
                 .pathMatchers("/groups/**").hasRole(viewGroupsRole)
                 .pathMatchers("/users/**").hasRole(viewUsersRole)
                 .pathMatchers("/*").denyAll()
-            .anyExchange().authenticated().and()
-            .oauth2ResourceServer().jwt()
-            .jwtAuthenticationConverter(new ReactiveJwtAuthenticationConverterAdapter(jwtAuthenticationConverter));
+                .anyExchange().authenticated()
+            .and()
+                .oauth2Client()
```

Comment:
I indented the code for clarity, but the new piece is the addition of the `oauth2Client`. In my testing it doesn't make a difference, but all of the examples I've seen include it. Maybe it only matters if you want to customize it, as we have with the `oauth2ResourceServer` and `authorizeExchange`.